lib/cli.js

```diff
@@ -234,7 +243,7 @@ const meowOptions /*: meowOptionsType*/ = {
 
       --formatter, -f               [default: "string"]
 
-        The output formatter: "compact", "json", "string" or "verbose".
+        The output formatter: ${formatterOptions}.
```

Comment:
To avoid a bug such as #3521, I change to generate a formatter options text from `formatters` object.
But there are no tests for help text, so I don't add test cases for this change.

Here is a partial help text related to this change, which is output in my CLI termial:

```
$ ./bin/stylelint.js -h | grep -C3 '\-\-formatter' 
      If the directory for the cache does not exist, make sure you add a trailing "/"
      on *nix systems or "\" on Windows. Otherwise the path will be assumed to be a file.

    --formatter, -f               [default: "string"]

      The output formatter: "compact", "json", "string", "unix" or "verbose".
```

This change may be over-engineering, so I want to hear opinions of reviewes about this change.
(I will go back to hard-coded text, if there are more oppositions)